File: SurgeonPortal.Api/ClientApp/src/app/api/models/professionalstanding/medical-license-read-only.model.ts

export interface IMedicalLicenseReadOnlyModel {
    licenseId: number;
    userId: number;
    issuingStateId: string;
    issuingState: string;
    licenseNumber: string;
    licenseTypeId: number;
    licenseType: string;
    issueDate: string;
    expireDate: string;
    reportingOrganization: string;
}

File: SurgeonPortal.Models/ProfessionalStanding/MedicalLicenseReadOnlyModel.cs
using System;

namespace SurgeonPortal.Models.ProfessionalStanding
{
    public class MedicalLicenseReadOnlyModel
    {
        public decimal LicenseId { get; set; }
        public int? UserId { get; set; }
        public string IssuingStateId { get; set; }
        public string IssuingState { get; set; }
        public string LicenseNumber { get; set; }
        public int? LicenseTypeId { get; set; }
        public string LicenseType { get; set; }
        public DateTime? IssueDate { get; set; }
        public DateTime? ExpireDate { get; set; }
        public string ReportingOrganization { get; set; }
    }
}
